themes/frontierline-research/includes/newsletter-form.php

```diff
@@ -0,0 +1,350 @@
+            <option value="sl">Sierra Leone</option>
+            <option value="sg">Singapore</option>
+            <option value="sx">Sint Maarten</option>
+            <option value="sk">Slovakia</option>
+            <option value="si">Slovenia</option>
+            <option value="sb">Solomon Islands</option>
+            <option value="so">Somalia</option>
+            <option value="za">South Africa</option>
+            <option value="gs">South Georgia and South Sandwich Islands</option>
+            <option value="ss">South Sudan</option>
+            <option value="es">Spain</option>
+            <option value="xs">Spratly Islands</option>
+            <option value="lk">Sri Lanka</option>
+            <option value="sd">Sudan</option>
+            <option value="sr">Suriname</option>
+            <option value="xr">Svalbard</option>
+            <option value="sz">Swaziland</option>
+            <option value="se">Sweden</option>
+            <option value="ch">Switzerland</option>
+            <option value="sy">Syria</option>
+            <option value="tw">Taiwan</option>
+            <option value="tj">Tajikistan</option>
+            <option value="tz">Tanzania</option>
+            <option value="th">Thailand</option>
+            <option value="tl">Timor-Leste</option>
+            <option value="tg">Togo</option>
+            <option value="tk">Tokelau</option>
+            <option value="to">Tonga</option>
+            <option value="tt">Trinidad and Tobago</option>
+            <option value="xt">Tromelin Island</option>
+            <option value="tn">Tunisia</option>
+            <option value="tr">Turkey</option>
+            <option value="tm">Turkmenistan</option>
+            <option value="tc">Turks and Caicos Islands</option>
+            <option value="tv">Tuvalu</option>
+            <option value="ug">Uganda</option>
+            <option value="ua">Ukraine</option>
+            <option value="ae">United Arab Emirates</option>
+            <option value="gb">United Kingdom</option>
+            <option value="us">United States</option>
+            <option value="uy">Uruguay</option>
+            <option value="uz">Uzbekistan</option>
+            <option value="vu">Vanuatu</option>
+            <option value="va">Vatican City</option>
+            <option value="ve">Venezuela</option>
+            <option value="vn">Vietnam</option>
+            <option value="vg">Virgin Islands, British</option>
+            <option value="vi">Virgin Islands, U.S.</option>
+            <option value="qw">Wake Island</option>
+            <option value="wf">Wallis and Futuna</option>
+            <option value="xw">West Bank</option>
+            <option value="eh">Western Sahara</option>
+            <option value="ye">Yemen</option>
+            <option value="zm">Zambia</option>
+            <option value="zw">Zimbabwe</option>
+          </select>
+        </div>
+
+        <div class="field field-language">
```

Comment:
The Labs newsletter is only available in English so this entire field should be removed.

Reply:
👍